### PR TITLE
Add git autoupdate config in dollar.js

### PR DESCRIPTION
**Pull request by LeaYeh**
@Amomo  Would you mind checking this PR for me?

### CODE REVIEW

**Comment by Amomo** [comment]
Hi @LeaYeh , in latest version, the content of `dollar.js` only contains 3 lines of code.
By reading original repo's package.json, I think we should also add `dollar/` directory.

**Comment by LeaYeh** [comment]
thank you pointed out my mistake which i never consider :)

**Comment by Amomo** [comment]
@LeaYeh  sorry!! I am late for checking this PR.
I think we should provide every files in dollar/ to satisfy usages on different platform.

**Comment by Amomo** [comment]
I think it is good now!
@PeterDaveHello could you take a look?

File: ajax/libs/dollar.js/package.json [hunk 1]
```diff
@@ -23,5 +23,14 @@
...
     "ozjs"
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dexteryy/DollarJS.git",
+    "basePath": "",
+    "files": [
+      "dollar*.js",
+      "dollar/**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/dollar.js/package.json]: I don't think we need it, and you should not use wildcard in non-dist folder ...

Comment (Amomo) [inline ajax/libs/dollar.js/package.json]: Don't we need the files in `dollar/` ?

Comment (PeterDaveHello) [inline ajax/libs/dollar.js/package.json]: Why do we need? I think this auto-update is not good at all, both line 32 and line 33.

**Comment by PeterDaveHello** [comment]
You should read the content in the files you want to add.

**Pull request closed without merging**